test/specs/app_test.js

```diff
@@ -1,151 +1,154 @@
+//     this.timeout(30000);
+
+//     beforeEach(() => {
+//         env.createTestUserDataDir();
+//         env.cleanTestConfig();
+//         this.app = env.getSpectronApp();
+//         return this.app.start();
+//     });
+
+//     afterEach(async () => {
+//         if (this.app && this.app.isRunning()) {
+//             await this.app.stop();
+//         }
+//     });
+
+//     // it('should show two windows if there is no config file', async () => {
+//     //     await this.app.client.waitUntilWindowLoaded();
+//     //     const count = await this.app.client.getWindowCount();
+//     //     count.should.equal(2);
+//     //     const opened = await this.app.browserWindow.isDevToolsOpened();
+//     //     opened.should.be.false;
+
+//     //     const visible = await this.app.browserWindow.isVisible();
+//     //     visible.should.be.true;
+//     // });
+
+//     if (process.platform === 'darwin') {
+//         it.skip('should show closed window with cmd+tab', async () => {
+//             // Unable to utilize Command key press due to: https://bugs.chromium.org/p/chromedriver/issues/detail?id=3023#c2
+//             await this.app.client.waitUntilWindowLoaded();
+//             await this.app.client.keys(['Meta', 'w']);
+//             let visible = await this.app.browserWindow.isVisible();
+//             visible.should.be.false;
+
+//             this.app.client.keys(['Meta', 'Tab']);
+//             visible = await this.app.browserWindow.isVisible();
+//             visible.should.be.true;
+//         });
+//     }
+
+//     it.skip('should restore window bounds', async () => {
+//         // bounds seems to be incorrectly calculated in some environments
+//         // - Windows 10: OK
+//         // - CircleCI: NG
+//         const expectedBounds = {x: 100, y: 200, width: 300, height: 400};
+//         fs.writeFileSync(env.boundsInfoPath, JSON.stringify(expectedBounds));
+//         await this.app.restart();
+//         const bounds = await this.app.browserWindow.getBounds();
+//         bounds.should.deep.equal(expectedBounds);
+//     });
+
+//     it('should NOT restore window bounds if the origin is located on outside of viewarea', async () => {
+//     // bounds seems to be incorrectly calculated in some environments (e.g. CircleCI)
+//     // - Windows 10: OK
+//     // - CircleCI: NG
+//         fs.writeFileSync(env.boundsInfoPath, JSON.stringify({x: -100000, y: 200, width: 300, height: 400}));
+//         await this.app.restart();
+//         let bounds = await this.app.browserWindow.getBounds();
+//         bounds.x.should.satisfy((x) => (x > -10000));
+
+//         fs.writeFileSync(env.boundsInfoPath, JSON.stringify({x: 100, y: 200000, width: 300, height: 400}));
+//         await this.app.restart();
+//         bounds = await this.app.browserWindow.getBounds();
+//         bounds.y.should.satisfy((y) => (y < 10000));
+//     });
+
+//     // it('should show settings.html when there is no config file', async () => {
```

Comment:
Yo dawg, I heard you liked commented out code...

Reply:
![image](https://user-images.githubusercontent.com/52460000/133270289-4d41b586-7b1a-4125-8edb-d8fd5f9ea68b.png)